index.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Porsche Cars</title>
+  <link rel="stylesheet" href="./css/styles.css">
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.2.0/css/all.css" />
+
+</head>
+
+<body>
+
+  <!-- HEADER SECTION STARTS -->
+  <section class="main-header">
+
+    <header class="header">
+      
+      <div class="top-navbar">
+        <div class="icons-n-text">
+          <a href="#" class="fab fa-facebook-f"></a>
+          <a href="#" class="fab fa-twitter"></a>
+          <a href="#" class="top-navbar-text">English</a>
+          <a href="#" class="top-navbar-text">My Page</a>
+          <a href="#" class="top-navbar-text">Logout</a>
+        </div>
+      </div>
+
+      <div class="second-navbar">
+
+          <a href="index.html" class="brand">
+            <h2 class="logo">Porsche</h2>
+          </a>
+
+          <nav class="navbar">
+
+            <ul class="nav-list">
+
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#program" class="nav-link">Program</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#join" class="nav-link">Join</a>
+              </li>
+              
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#news" class="nav-link">News</a>
+              </li>
+              
+              <li class="nav-item">
+                <a href="#" class="nav-link-cta">Porsche Cars</a>
+              </li>
+
+            </ul>
+
+          </nav>
+
+          <div class="icons">
+            <div class="fas fa-bars" id="menu-btn"></div>
+          </div>
+
+      </div>
+
+    </header>
+
+  </section>
+
+  <!-- HERO SECTION STARTS -->
+
+  <section class="hero">
+    <div class="hero-content">
+      <h5>Class Is Permanent</h5>
+      <h1 class="hero-title">We are the Porsche Club of America</h1>
+      <p class="hero-para">The Porsche Club of America is the largest single-marque car club in the world.</p>
+      <h3 class="hero-date"> 2022.11.24 (THU) ~ 28 (FRI)</h3>
+      <p class="hero-venue">
+        @ National Harbor, Maryland, USA
+      </p>
+    </div>
```

Comment:
- [ ] Great job on using the right smeantic tags here, kindly try to style the headline according to the guidelines, this includes `font-size`, `font-weight` and text `color` 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/203852392-5b9ff81d-10ca-45fc-9368-c7d4c5cd0945.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/203852417-69ae6a1e-1596-45bc-aeb8-e4d2c7038108.png)